src/components/select/select.js

```diff
@@ -69,6 +70,18 @@ angular.module('material.components.select', [
  *   </md-input-container>
  * </hljs>
  *
+ * With a select-header
```

Comment:
Can you add a small paragraph here describing potential use-cases and what happens to the original label?